src/EditorFeatures/Test/MetadataAsSource/MetadataAsSourceTests.cs

```diff
@@ -37,6 +40,26 @@ public class [|C|]
 End Class");
         }
 
+        [WorkItem(4019, "https://github.com/dotnet/roslyn/issues/4019")]
+        [Fact, Trait(Traits.Feature, Traits.Features.MetadataAsSource)]
```

Comment:
Given this isn't a bug in MetadataAsSource per se, can't we just test this with formatting tests? I recognize no user-inputted text will replicate it, but we really should have tests for this scenario directly.

(I'd prefer there actually aren't any tests for MAS here.)